diff -crB sglang_old/srt/model_executor/model_runner.py sglang/srt/model_executor/model_runner.py
*** sglang_old/srt/model_executor/model_runner.py	Thu Jun 26 05:02:09 2025
--- sglang/srt/model_executor/model_runner.py	Thu Jun 26 05:02:50 2025
***************
*** 631,637 ****
                  rank=rank,
                  group_name=group_name,
              )
-             dist.barrier(group=self._model_update_group, device_ids=[rank])
              return True, "Succeeded to initialize custom process group."
          except Exception as e:
              message = f"Failed to initialize custom process group: {e}."
--- 631,636 ----
